load-code/pe/copy-pe-sections.yml

```diff
@@ -0,0 +1,32 @@
+rule:
+  meta:
+    name: copy PE sections
+    namespace: load-code/pe
+    author: "@Ana06"
+    scope: function
+    references:
+      - https://0x00sec.org/t/reflective-dll-injection/3080
+      - https://www.ired.team/offensive-security/code-injection-process-injection/reflective-dll-injection
+    examples:
+      - E4C33AC3638EEF68311F8AC0D72483C7:0x401510
+  features:
+    - and:
+      - offset: 0x6 = IMAGE_NT_HEADERS.FileHeader.NumberOfSections
```

Comment:
could we pull a bunch of this logic into "enumerate pe sections"?

Reply:
are you proposing to rename this rule and remove `- mnemonic: call = call CopyMemory, memcpy or similar function`? 😕

Reply:
im proposing to factor out the section looping into its own rule, and have this rule `match: enumerate PE sections`

Reply:
Discussed offline to remove `mnemonic: call = call CopyMemory, memcpy or similar function` and call this rule `enumerate PE sections`